src/app/pages/confirm-test-details/components/confirm-submit-modal/confirm-submit-modal.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ModalController,
+} from '@ionic/angular';
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'confirm-submit-modal',
+  templateUrl: './confirm-submit-modal.html',
+  styleUrls: ['./confirm-submit-modal.scss'],
+})
+export class ConfirmSubmitModal {
+
+  onTestDetailsConfirm: Function;
+  @Input('testOutcome') testOutcome;
+  @Input('category') category;
+  @Input('candidateName') candidateName;
```

Comment:
Are we able to add some appropriate types to these inputs?

Reply:
Not sure the inputs are nessaccery here.  this seems to work fine.  However agree that types are needed.
![image](https://user-images.githubusercontent.com/48550267/135411598-250f1ed9-1e03-48fe-9ba9-7d699dbde43c.png)